LINQTasks/GettingLastWord.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace LINQTasks
+{
+    public class GettingLastWord
+    {
+        public static string GetTheLastWord(IEnumerable<string> words)
```

Comment:
The method's name doesn't reflect/mentions anything related to filtering words that have e letter